tests/integration/backward_compatible/proxy/multi_map_test.py

```diff
@@ -150,6 +150,14 @@ def test_put_get(self):
 
         self.assertCountEqual(self.multi_map.get("key"), ["value1", "value2"])
 
+    def test_put_all_get(self):
+        skip_if_client_version_older_than(self, "5.2")
```

Comment:
I see that the multimap put all is added in server version 4.1. there should be a server version check with this as well